index.js

```diff
@@ -214,7 +205,7 @@ module.exports = class Hypercore extends EventEmitter {
     this.key = o.key
     this.core = o.core
     this.replicator = o.replicator
-    this.encryption = o.encryption
+    if (o.encryption) this.encryption = o.encryption
```

Comment:
does the if here matter?

Reply:
No, sorry, that was a silly attempt to protect the original core from being overridden.

Reply:
np, let's remove it then